tests/chainer_tests/links_tests/connection_tests/test_tree_lstm.py

```diff
@@ -97,7 +107,7 @@ def _nary_tree_lstm(func, *inputs):
 
 @testing.parameterize(*testing.product({
     'dtype': [numpy.float32],
-    'n_ary': [2, 3],
+    'n_ary': [0, 1, 2, 3],
```

Comment:
Why do you add `0`?

Reply:
ChildSumTreeLSTM can accept variable-length children as input. And, at leaf nodes, the number of children is `0`.
In that case, I think it is more useful to call `func(x)` directly than `func(None, None, x)`, where `x` is the input value of the node. This `0`-ary test is prepared for the call.

Reply:
ChildSumTreeLSTM can accept variable-length children as input. And, at leaf nodes, the number of children is `0`.
In that case, I think it is more useful to call `func(x)` directly, where x is the input value of the node. This `0`-ary test is prepared for the call.

Reply:
OK